### DISCUSSION TITLE
How to remove white circle with play triangle when paused to read text in video?

### DISCUSSION THREAD

**Discussion by emw1086** [FreeTubeApp/FreeTube#6268] [Q&A] [answered]
Hello,

Is there any way to remove the white circle with the black play triangle when a video is paused?  There are many times where I want to pause a video to have more time to read a slide or other text but it is obscured by the play button in the middle of the screen and sometimes by the slider at the bottom.  

I thought in older versions that these faded out after a few seconds or maybe it was when you moved the mouse off the window but nothing seems to work now.  I looked through all the settings but do not see anything related.  Am I missing something?  Is there a way to get it to do this (setting option, mouse cursor movement, key press, etc.) ?

Thanks!
-Eric

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment, marked as answer]
duplicate of https://github.com/FreeTubeApp/FreeTube/discussions/6107

**Comment by emw1086** [comment]
Thanks for your help!
